package.json

```diff
@@ -17,7 +17,9 @@
     "typescript": "^5.2.2"
   },
   "scripts": {
+    "gdpr": "node scripts/gdpr-generate.js",
```

Comment:
I have added the generated json-s in #2 

the other solution could be to add these files to .gitignore